dhall-toml/src/Dhall/Toml.hs

```diff
@@ -1,35 +1,38 @@
-{-# LANGUAGE OverloadedStrings #-}
-
 module Dhall.Toml
-    ( tomlToDhall
+    ( tomlToDhallMain
+    , tomlToDhall
```

Comment:
I think these functions should eventually be moved to a separate module (e.g. `Dhall.TomlToDhall`) but this doesn't necessarily have to happen now.

Reply:
Okay, I'll add a "TODO" for that